### DISCUSSION TITLE
Closing the symbol searching window properly

### DISCUSSION THREAD

**Discussion by AlphabetsAlphabets** [neoclide/coc.nvim#3145] [Q&A] [answered]
So COC has a keybinding that opens up a new window at the bottom and lets you fuzzy find symbols throughout your project, the problem is sometimes I hit the keybinding by accident, and the window shows up. The problem is with closing, I can't find any other way to close it other than with control+c but closing the window that way causes your cursor to disappear. It remains missing even after exiting vim. I can still type within vim and the terminal but the cursor is just gone.
`nnoremap <silent><nowait> <space>S  :<C-u>CocList -I symbols<cr>` this is the keybinding for anyone wondering.

**Comment by chemzqm** [comment, marked as answer]
Use `<Esc>` instead of `<C-c>` which can't be detected.

**Reply by AlphabetsAlphabets** [reply]
Using `<Esc>` doesn't work. Is there a way to bind this to a different key?

**Reply by chemzqm** [reply]
`<esc>` should always work, can't reproduce your problem.
Checkout `:h coc-list-mappings`

**Reply by AlphabetsAlphabets** [reply]
Does it have anything to do with the fact I mapped `ii` to have the same function as `<Esc>`?
